server/sonar-server/src/main/java/org/sonar/server/exceptions/Errors.java

```diff
@@ -70,20 +70,12 @@ public boolean check(boolean expression, String l10nKey, Object... l10nParams) {
     return expression;
   }
 
-  public void writeJson(JsonWriter json, I18n i18n, Locale locale) {
-    writeJson(json, i18n, locale, "errors");
-  }
-
-  public void writeJsonAsWarnings(JsonWriter json, I18n i18n, Locale locale) {
-    writeJson(json, i18n, locale, "warnings");
-  }
-
-  private void writeJson(JsonWriter json, I18n i18n, Locale locale, String name) {
+  public void writeJson(JsonWriter json, I18n i18n) {
     if (!messages.isEmpty()) {
-      json.name(name).beginArray();
+      json.name("errors").beginArray();
       for (Message message : messages) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "Message"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)